lsat/satisfier.go

```diff
@@ -115,3 +119,64 @@ func NewCapabilitiesSatisfier(service string, targetCapability string) Satisfier
 		},
 	}
 }
+
+// NewTimeoutSatisfier checks if an LSAT is expired or not. The Satisfier takes
```

Comment:
previous comment resolved but not addressed yet - pls wrap things at 80 chars :)